src/Common/src/System/FormattingHelpers.cs

```diff
@@ -2,10 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
 using System.Globalization;
 
-namespace ILCompiler
+namespace System
```

Comment:
Nit: I know putting this in the System namespace was my idea, but maybe putting it into the toplevel namespace would be less of a misuse?